### ISSUE TITLE
inherit scaling from parent widgets

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#102]
widgets that don't specify their own scale should be allowed to inherit scaling from their parents

**Comment by HeladoDeBrownie** [comment]
this likely means mixing `Scalable` at the `Widget` level and then doing a fallback similar to how color schemes are done
